js/apps/admin-ui/src/realm-settings/RealmSettingsTabs.tsx

```diff
@@ -235,6 +235,34 @@ export const RealmSettingsTabs = () => {
         ),
       );
     }
+    if (r.attributes) {
+      // Handle nested vc object created by React Hook Form
+      if (r.attributes["vc"] && typeof r.attributes["vc"] === "object") {
```

Comment:
Instead of doing this after the fact, we usually flatten the keys to something when registering the form controls. See `convertAttributeNameToForm()`.